test/k8sT/Services.go

```diff
@@ -450,11 +450,13 @@ var _ = Describe("K8sServicesTest", func() {
 		// dstPodPort:  Receiver pod port (for checking in CT table)
 		// dstIP:       Target endpoint IP for sending the datagram
 		// dstPort:     Target endpoint port for sending the datagram
-		doFragmentedRequest := func(srcPod string, srcIP string, fromNode string, dstPodPort int, dstIP string, dstPort int32) {
+		// kubeProxy:   True if kube-proxy is enabled
+		doFragmentedRequest := func(srcPod, srcIP, fromNode string, dstPodPort int, dstIP string, dstPort int32, kubeProxy bool) {
 			var (
 				blockSize  = 5120
 				blockCount = 1
 				srcPort    = 12345
+				hostIP     = "10.0.2.15"
```

Comment:
We can't use that method in tests, can we?

I don't like this either. I looked at using e.g., `GetNodeIPByLabel(..., true)`, but it wasn't returning anything (the `ExternalIP` field didn't exist in kubectl's JSON output). This seems expected since other tests are also hardcoding the IP:
```
$ git grep 10.0.2.15
k8sT/Services.go:                               hostIP     = "10.0.2.15"
runtime/Policies.go:            hostIP := "10.0.2.15"
runtime/Policies.go:                    endpointID, _ := createEndpoint("ping", "10.0.2.15")
runtime/Policies.go:                            endpointID, _ := createEndpoint("ping", "10.0.2.15")
runtime/Policies.go:                    hostIP := "10.0.2.15"
```
At the very least, we could move this value to some shared file, like `helpers.K8s1IP`.

Reply:
I'm good with moving it to a helper, let's not spend too much on this. We can improve in the future if necessary, or maybe someone else will review and have an idea on how to get it in a clean way.